apis/enigma-mocker/README.md

```diff
@@ -0,0 +1,86 @@
+# @nebula.js/enigma-mocker
+
+The purpose of the Enigma mocker is to be able to render charts without a connected engine. This is useful for example when running rendering tests or in code examples.
```

Comment:
```suggestion
The purpose of the Enigma mocker is to be able to render charts without a connected Qlik engine. This could be useful for example when running rendering tests or in code examples.
```

Reply:
Nice one! I've incorporated these changes in the latest and greatest version.